src/brs/BlockchainProcessorImpl.java

```diff
@@ -1166,7 +1167,7 @@ public void generateBlock(String secretPhrase, byte[] publicKey, Long nonce) thr
             slotToTakeFrom.remove(highestPriority.get());
           });
           transactionsToBeIncluded = unconfirmedTransactionsOrderedBySlot;
-        } else { // Before Pre-Dymaxion HF, just choose highest priority
+        } else { // Before Pre-POC2 HF, just choose highest priority
```

Comment:
Removed all mentions to dymaxion, since it didn't happen this name makes no sense.